pom.xml

```diff
@@ -101,6 +101,7 @@
           <charset>UTF-8</charset>
           <docencoding>UTF-8</docencoding>
           <encoding>UTF-8</encoding>
+          <additionalparam>--allow-script-in-comments</additionalparam>
```

Comment:
What's this about?

Reply:
That is required to make java doc run, which in this package is required to build a jar. If you're not happy with script in comments, I could go through the code and try to find out what javadoc thinks are the script comments.